cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -326,6 +341,130 @@ def upload_rna_coverage_bigwig_to_scout(self, case_id: str, dry_run: bool) -> No
+            rna_sample_internal_id: str = rna_dna_collection.rna_sample_internal_id
+            dna_sample_name: str = rna_dna_collection.dna_sample_name
+
+            for dna_case_id in rna_dna_collection.dna_case_ids:
+                LOG.info(
+                    f"Uploading omics sample id for sample {dna_sample_name} "
+                    f"in case {dna_case_id} in Scout."
+                )
+
+                if dry_run:
+                    continue
+
+                self.scout_api.upload_omics_sample_id(
+                    dna_case_id=dna_case_id,
+                    customer_sample_id=dna_sample_name,
+                    rna_sample_internal_id=rna_sample_internal_id,
+                )
+
+    def upload_rna_fraser_outrider_to_scout(self, dry_run: bool, case_id: str) -> None:
+        """Upload omics fraser and outrider file for a case to Scout."""
+        status_db: Store = self.status_db
+        rna_case = status_db.get_case_by_internal_id(case_id)
+        cust_id = rna_case.customer.internal_id
+        rna_dna_collections: list[RNADNACollection] = self.create_rna_dna_collections(rna_case)
+        for rna_dna_collection in rna_dna_collections:
+            rna_sample_internal_id: str = rna_dna_collection.rna_sample_internal_id
+            dna_sample_name: str = rna_dna_collection.dna_sample_name
+            rna_fraser: File | None = self.get_rna_omics_fraser(case_id=case_id)
+            rna_outrider: File | None = self.get_rna_omics_outrider(case_id=case_id)
+            if not rna_fraser:
+                raise FileNotFoundError(
+                    f"No RNA fraser file was found in housekeeper for {rna_sample_internal_id}."
+                )
+            if not rna_outrider:
+                raise FileNotFoundError(
+                    f"No RNA outrider file was found in housekeeper for {rna_sample_internal_id}."
+                )
+
+            LOG.debug(f"RNA fraser file {rna_fraser.path} found.")
+            LOG.debug(f"RNA outrider file {rna_outrider.path} found.")
+
+            for dna_case_id in rna_dna_collection.dna_case_ids:
+                LOG.info(
+                    f"Uploading RNA fraser and outrider files for sample {dna_sample_name} "
+                    f"in case {dna_case_id} in Scout."
+                )
+                customer_case = status_db.get_case_by_internal_id(dna_case_id)
+
+                if dry_run:
+                    continue
+
+                self.scout_api.upload_rna_fraser_outrider(
+                    fraser_file_path=rna_fraser.full_path,
+                    outrider_file_path=rna_outrider.full_path,
+                    case_id=dna_case_id,
+                    customer_case_name=customer_case.name,
+                    cust_id=cust_id,
+                )
+
+        for upload_statement in self.get_rna_fraser_outrider_upload_summary(rna_dna_collections):
+            LOG.info(upload_statement)
+        LOG.info("Upload RNA fraser and outrider file finished!")
```

Comment:
I see a lot of logging going on here but I also saw similar information being logged in the scoutapi. Maybe a lot of these can be removed in favour of the other ones.

Imagine a RNA-case going through the whole flow, how would the logs look currently and would all the messges be informative?

Reply:
I would keep the logging as similar as mip-rna before we refactor